bok_choy/javascript.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable= literal-used-as-attribute
```

Comment:
Instead of disabling this for the whole file, I think it's better to disable this on the line where the pylint error was showing up, so we know that we are only doing it for those specific cases.